### PR TITLE
Removed React context mention in the documentation

### PR DESCRIPTION
**Pull request by maoueh**
In the section `Translate your own components`

### CODE REVIEW

**Comment by maoueh** [comment]
Replaces #2829

File: docs/Translation.md [hunk 1]
```diff
@@ -279,36 +279,7 @@ const App = () => (
...
-            hello: {
-                world: 'Hello, World!',
-            },
-        },
-    },
-};
-```
-
-However, using the context makes components harder to test. That's why react-admin provides a `translate` Higher-Order Component, which simply passes the `translate` function from context to props:
+The react-admin package provides a `translate` Higher-Order Component, which simply passes the `translate` function to props of wrapped component:
```
Comment (djhi) [inline docs/Translation.md]:
```suggestion
React-admin provides a `translate` Higher-Order Component, which simply inject the `translate` function to the wrapped component:
```

File: docs/Translation.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ title: "Translation"
 
 # Translation
 
-The react-admin interface uses English as the default language. But it also supports any other language, thanks to the [polyglot.js](http://airbnb.io/polyglot.js/) library.
+React-admin interface uses English as the default language. But it also supports any other language, thanks to the [polyglot.js](http://airbnb.io/polyglot.js/) library.
```
Comment (djhi) [inline docs/Translation.md]: This one should have kept the `The` as we are refering to the interface :wink:

File: docs/Translation.md [hunk 3]
```diff
@@ -279,36 +279,7 @@ const App = () => (
...
-            hello: {
-                world: 'Hello, World!',
-            },
-        },
-    },
-};
-```
-
-However, using the context makes components harder to test. That's why react-admin provides a `translate` Higher-Order Component, which simply passes the `translate` function from context to props:
+React-admin package provides a `translate` Higher-Order Component, which simply passes the `translate` function to props of the wrapped component:
```
Comment (djhi) [inline docs/Translation.md]: Last one I think :smile: 

s/to props of/as a prop to/

Comment (maoueh) [reply docs/Translation.md]: Went really too quick :)

**Pull request merged**